webviz_subsurface/plugins/_reservoir_simulation_timeseries_onebyone.py

```diff
@@ -68,14 +68,68 @@ class ReservoirSimulationTimeSeriesOneByOne(WebvizPluginABC):
     ]
 
     TABLE_STAT = [
-        "Sensitivity",
-        "Case",
-        "Mean",
-        "Standard Deviation",
-        "Minimum",
-        "P90",
-        "P10",
-        "Maximum",
+        ("Sensitivity", {}),
+        ("Case", {}),
+        (
+            "Mean",
+            {
+                "type": "numeric",
+                "format": {
+                    "locale": {"symbol": ["", "unit_insert"]},
+                    "specifier": "$.4s",
+                },
+            },
+        ),
+        (
+            "Stddev",
+            {
+                "type": "numeric",
+                "format": {
+                    "locale": {"symbol": ["", "unit_insert"]},
+                    "specifier": "$.4s",
+                },
+            },
+        ),
+        (
+            "Minimum",
+            {
+                "type": "numeric",
+                "format": {
+                    "locale": {"symbol": ["", "unit_insert"]},
+                    "specifier": "$.4s",
+                },
+            },
+        ),
+        (
+            "P90",
+            {
+                "type": "numeric",
+                "format": {
+                    "locale": {"symbol": ["", "unit_insert"]},
+                    "specifier": "$.4s",
+                },
+            },
+        ),
+        (
+            "P10",
+            {
+                "type": "numeric",
+                "format": {
+                    "locale": {"symbol": ["", "unit_insert"]},
+                    "specifier": "$.4s",
+                },
+            },
+        ),
+        (
+            "Maximum",
+            {
+                "type": "numeric",
+                "format": {
+                    "locale": {"symbol": ["", "unit_insert"]},
+                    "specifier": "$.4s",
+                },
+            },
+        ),
```

Comment:
Since several plugins use this, should we move it to a centralized location?

Reply:
Is a trade-off between simplicity and configurability, but yes might be a good idea to move it. If necessary in a plugin it could be overwritten locally. Could it fit in the abbreviations/number_formatting.py?

Reply:
Yes, i think that's a good place 👍

Reply:
done